File: easy/array/Minimum Number of Moves to seat everyone-2037-easy.js
/**
 * @param {number[]} seats
 * @param {number[]} students
 * @return {number}
 */
var minMovesToSeat = function(seats, students) {
    let sum = 0 ;
    seats = seats.sort((a,b)=> a-b);    
    students = students.sort((a,b)=> a-b);

    for(let i=0; i<seats.length; i++){
        sum += Math.abs(seats[i] - students[i]);
    }    
    return sum;
};
const result = minMovesToSeat([3,1,5], [2,7,4]);
console.log(result);

File: Fundamental/Sorting/Minimum Number of Moves to seat everyone-2037-easy.py
def min_moves_to_seat(seats, students):
    total_moves = 0
    seats.sort()
    students.sort()

    for i in range(len(seats)):
        total_moves += abs(seats[i] - students[i])

    return total_moves

result = min_moves_to_seat([3, 1, 5], [2, 7, 4])
print(result)
